march_rqt_gait_selection/src/march_rqt_gait_selection/parametric_pop_up.py

```diff
@@ -0,0 +1,78 @@
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QComboBox, QDialog, QDialogButtonBox, QFormLayout, QGridLayout, QLabel, QLineEdit,\
+    QMessageBox
+
+
+class ParametricPopUpWindow(QDialog):
```

Comment:
You could also create a `.ui` file for this, just like the main widget and load it.